.github/workflows/deploy-to-production.yml

```
@@ -80,26 +90,28 @@
           images: ${{ env.REGISTRY_IMAGE }}
+          tags: ${{ env.REGISTRY_IMAGE }}:${{ matrix.tag }}
 
       - name: Log in to Docker Hub 🔐
         uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_PASSWORD }}
 
-      - name: Create and push manifest list 📦
+      - name: Create manifest list and push 📦
         run: |
```

actionlint

property "tag" is not defined in object type {}

(expression)